### ISSUE TITLE
InteractBlockEvent triggers on placing blocks, should we allow this behaviour?

### ISSUE THREAD

**Issue by Admicos** [okyanus-mc/runtime#6]

**Comment by portlek** [comment]
interact events can be abstract, and we can add BlockClick, BlockBreak, BlockExplosion etc. events.

**Comment by Admicos** [comment]
No, that's not what I meant. We already have a BreakEvent and PlaceEvent. It's just InteractBlockEvent (right clicking on blocks) also trigger on block placing.

**Comment by portlek** [comment]
you need to right click to place a block, i think we should.
